server/src/main/java/org/opensearch/action/admin/indices/mapping/put/PutMappingRequest.java

```diff
@@ -79,7 +79,7 @@
  */
 public class PutMappingRequest extends AcknowledgedRequest<PutMappingRequest> implements IndicesRequest.Replaceable, ToXContentObject {
 
-    private static ObjectHashSet<String> RESERVED_FIELDS = ObjectHashSet.from(
+    private static Set<String> RESERVED_FIELDS = Set.of(
```

Comment:
Maybe add `final`, since you're here. Let's hope this constant isn't being reassigned, anyway :)